tests/snapshots/test_cli__long-help.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: tests/test-cli.rs
+assertion_line: 104
```

Comment:
I'm a bit surprised these lines are showing up in the yaml - is this a change with newer versions of `cargo insta`? IIRC they should only be in `.snap.new` files.

Reply:
Fixed. I had just copied the .snap.new files because I didn't see the instruction to run `cargo insta review`